src/components/SavedNewsList/SavedNewsList.js

```diff
@@ -0,0 +1,41 @@
+import { useContext, useEffect } from "react";
+
+import "../SavedNews/SavedNews.css";
+import NewsCard from "../NewsCard/NewsCard";
+import { CurrentUserContext } from "../../contexts/CurrentUserContext";
+import { SavedNewsContext } from "../../contexts/SavedNewsContext";
+import { getSavedNews } from "../../utils/api";
+
+const SavedNewsList = ({
+  handleSaveCard,
+  handleDeleteCard,
+  onLogin,
+  isSaved,
+}) => {
+  const { currentUser } = useContext(CurrentUserContext);
+  const { savedNews, setSavedNews } = useContext(SavedNewsContext);
+
+  useEffect(() => {
+    const jwt = localStorage.getItem("token");
+    getSavedNews(jwt).then(setSavedNews);
+  }, [setSavedNews]);
```

Comment:
` [setSavedNews]);`


* When I visit the **Saved articles** page, there are a lot of server requets https://snipboard.io/bBZi4d.jpg 


Please, use only **empty** array here because `setSavedNews` makes the effect trigger every second